jerry-core/jerry-snapshot.c

```diff
@@ -968,43 +973,43 @@ jerry_parse_and_save_literals (const jerry_char_t *source_p, /**< script source
   if (is_c_format)
   {
     /* Save literal count. */
-    buffer_p = jerry_append_chars_to_buffer (buffer_p,
-                                             buffer_end_p,
-                                             (const char *) "jerry_length_t literal_count = ",
-                                             0);
+    buffer_data_p = jerry_append_chars_to_buffer (buffer_data_p,
```

Comment:
Buffer data sounds like data data, a word repeat to me. What about destination_p?